.env.example

```diff
@@ -26,6 +26,8 @@ MAIL_PORT=2525
 MAIL_USERNAME=null
 MAIL_PASSWORD=null
 MAIL_ENCRYPTION=null
+MAIL_FROM_ADDRESS=regis@monicahq.com
```

Comment:
The problem is that people will copy this example file and barely touch the default if they don't have to. By removing MY info entirely, we are sure this won't pop up in my email ever.